docs/manual/java/guide/build/EclipseRunConfig.md

```diff
@@ -0,0 +1,21 @@
+# Creating an Eclipse Run Configuration
```

Comment:
Added Maven to title, and added Ignasi's tip to the appropriate topics. Once this was done, the topic on creating a runtime config was redundant, so I removed it. Now have to remove images. Will do this in the next commit